### DISCUSSION TITLE
A new idea for down line with servers

### DISCUSSION THREAD

**Discussion by Scorpions95** [Eight-P/BetterDiscord-Stuff#27] [Ideas]
I think it would be a good idea to make the line with the servers (the one at the bottom where it is) make it raise an arrow.

**Comment by Eight-P** [comment]
I don't think I understand what you're trying to say.
If you are looking for a way to hide the Serverbar you could try CollapsibleUI (even has Keybinds).
https://betterdiscord.app/plugin/CollapsibleUI

**Reply by Scorpions95** [reply]
![Screenshot 2024-04-09 19 24 21](https://github.com/Eight-P/BD.8P/assets/24534657/a5d8b4f3-2573-4aca-855d-1bf2cbc4fa1e)
i want this server line to be collapsed also i download plugin u provide me, but seems doesnt work

**Reply by Eight-P** [reply]
Well, that is exactly one of the things CollapsibleUI does.
There was a minor issue I just patched. Other than that, the plugin works just fine.
You hover over the memberslist icon, and it shows you additional icons to toggle other UI elements.
You can also do [Alt + S] for the server bar (on an English keyboard at least).
Check the GIF on the plugin's page, it shows you how to use it.
